### ISSUE TITLE
ui: db console displays "Insecure mode" when running in secure mode

### ISSUE THREAD

**Issue by ericharmeling** [cockroachdb/cockroach#88565]
The DB Console is showing "Insecure mode" in 22.2 testing releases running in secure mode, with secure connections to the DB Console.

Locally:

```
cockroach-22-2 start-single-node --certs-dir=$HOME/certs --listen-addr=localhost:26257 --http-addr=localhost:8080
*
* WARNING: Running a server without --sql-addr, with a combined RPC/SQL listener, is deprecated.
* This feature will be removed in the next version of CockroachDB.
*
CockroachDB node starting at 2022-09-23 14:13:08.671247 +0000 UTC m=+6.291288293 (took 5.8s)
build:               CCL v22.2.0-alpha.4 @ 2022/09/22 17:11:35 (go1.19.1)
webui:               https://localhost:8080
sql:                 postgresql://root@localhost:26257/defaultdb?sslcert=%2FUsers%2Fericharmeling%2Fcerts%2Fclient.root.crt&sslkey=%2FUsers%2Fericharmeling%2Fcerts%2Fclient.root.key&sslmode=verify-full&sslrootcert=%2FUsers%2Fericharmeling%2Fcerts%2Fca.crt
sql (JDBC):          jdbc:postgresql://localhost:26257/defaultdb?sslcert=%2FUsers%2Fericharmeling%2Fcerts%2Fclient.root.crt&sslkey=%2FUsers%2Fericharmeling%2Fcerts%2Fclient.root.key&sslmode=verify-full&sslrootcert=%2FUsers%2Fericharmeling%2Fcerts%2Fca.crt&user=root
RPC client flags:    cockroach-22-2 <client cmd> --host=localhost:26257 --certs-dir=/Users/ericharmeling/certs
logs:                /Users/ericharmeling/Downloads/cockroach-data/logs
temp dir:            /Users/ericharmeling/Downloads/cockroach-data/cockroach-temp2026111368
external I/O path:   /Users/ericharmeling/Downloads/cockroach-data/extern
store[0]:            path=/Users/ericharmeling/Downloads/cockroach-data
storage engine:      pebble
clusterID:           18cdc24a-909c-4f66-aa66-a32633b038ed
status:              initialized new cluster
nodeID:              1
```

![image](https://user-images.githubusercontent.com/27286675/191980837-69dc2976-4384-4379-85f9-df19fa747c02.png)

Cloud:

![image](https://user-images.githubusercontent.com/27286675/191979471-7dae6993-02ba-43cd-ae59-cf182d480e8c.png)

This behavior is not present in 22.1:

![image](https://user-images.githubusercontent.com/27286675/191982582-e550f4d6-1470-43b9-b978-69cc36893503.png)

Jira issue: CRDB-19944

Epic CRDB-21265

**Comment by NOBLES5E** [comment]
This bug is also found in the official 22.2.0 release.

**Comment by rafiss** [comment]
This appears to be fixed in v22.2.5
